src/codeu/chat/server/Model.java

```diff
@@ -68,9 +74,95 @@ public int compare(Time a, Time b) {
   private final Uuid.Generator userGenerations = new LinearUuidGenerator(null, 1, Integer.MAX_VALUE);
   private Uuid currentUserGeneration = userGenerations.make();
 
+  private final String database = "codeutest.db";
```

Comment:
Make these constants static and use CONSTANT_CASE: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names